### DISCUSSION TITLE
The github.dev web-based editor can't sync repo

### DISCUSSION THREAD

**Discussion by DavidSongzw** [community/community#8168] [Codespaces] [answered]
Because my repo is editing and updating, the web editor is also editing at the same time.which has some effect, so even if I reset the web editor, I still can't sync.The solution I eventually found was to clear the browser cache and reopen the web editor.
So it seems that the web editor is using the browser cache to store git-related files, which brings up the problem of local cache file errors that can't be synchronized, and I think the GitHub team could use a solution that gives users the option to drop the local cache file and pull the repo again if the local cache file is wrong.

**Comment by davorpa** [comment, marked as answer]
I find more or less the same issue with the browsers cache but starting the repo synced (new fresh branch). Steps:

- Enable search indexation engine
- Make changes using file view. Save them
- [Optional] Close editor and open again or use other browser tab
- Make some several changes from file view or source control and/or closing files.
- Commit
- Open one of those files again. Sometimes have the content of the previous version, file search cheat alternate depending of what text are looking for.

I haven't catch very well but also happens sometimes after clean local stores and databases

PROBLEM: This can lead to commits with the wrong code
